src/assets/Generator.Shared/Utf8JsonWriterExtensions.cs

```diff
@@ -140,5 +141,53 @@ public static void WriteObjectValue(this Utf8JsonWriter writer, object value)
                     throw new NotSupportedException("Not supported type " + value.GetType());
             }
         }
+
+        public static void WriteBinaryDataValue(this Utf8JsonWriter writer, BinaryData value)
+        {
+            byte[] inputValue = value.ToArray();
```

Comment:
Does this create a copy?  If so is there a way to access the raw bytes without a copy?

Reply:
Change to use ReadOnlySpan<byte> to avoid copying